geomstats/geometry/_hyperbolic.py

```diff
@@ -46,71 +46,71 @@ def _extrinsic_to_extrinsic_coordinates(point):
         Parameters
         ----------
         point : array-like, shape=[..., dim + 1]
-            Point in hyperbolic space in extrinsic coordinates.
+            Point in hyperbolic space in extrinsic-hyperboloidcoordinates.
 
         Returns
         -------
-        point_extrinsic : array-like, shape=[..., dim]
-            Point in hyperbolic space in extrinsic coordinates.
+        point_extrinsic-hyperboloid: array-like, shape=[..., dim]
+            Point in hyperbolic space in extrinsic-hyperboloidcoordinates.
         """
         return point
 
     @staticmethod
-    def _intrinsic_to_extrinsic_coordinates(point):
-        """Convert intrinsic to extrinsic coordinates.
+    def _intrinsic-hyperboloid_to_extrinsic_coordinates(point):
+        """Convert intrinsic-hyperboloid to extrinsic-hyperboloidcoordinates.
 
         Convert the parameterization of a point in hyperbolic space
-        from its intrinsic coordinates, to its extrinsic coordinates
+        from its intrinsic-hyperboloid coordinates, to its extrinsic-hyperboloidcoordinates
         in Minkowski space.
 
         Parameters
         ----------
         point : array-like, shape=[..., dim]
-            Point in hyperbolic space in intrinsic coordinates.
+            Point in hyperbolic space in intrinsic-hyperboloid coordinates.
 
         Returns
         -------
-        point_extrinsic : array-like, shape=[..., dim + 1]
-            Point in hyperbolic space in extrinsic coordinates.
+        point_extrinsic-hyperboloid: array-like, shape=[..., dim + 1]
+            Point in hyperbolic space in extrinsic-hyperboloidcoordinates.
         """
         coord_0 = gs.sqrt(1.0 + gs.sum(point**2, axis=-1))
-        point_extrinsic = gs.concatenate([coord_0[..., None], point], axis=-1)
+        point_extrinsic-hyperboloid= gs.concatenate([coord_0[..., None], point], axis=-1)
```

Comment:
Thanks for the contribution @soma2000-lang.

Notice that `point_extrinsic-hyperboloid` is not a valid python name (that's the reason for `_`).

(This happens in other places in the code)